src/batchnorm/problem_description.cpp

```diff
@@ -37,6 +37,17 @@ namespace miopen {
 namespace batchnorm {
 
 NetworkConfig ProblemDescription::MakeNetworkConfig() const
+{
+    switch(direction)
+    {
```

Comment:
Why forward changes in backward solver changes?

Reply:
This PR does not change the Fwd BN solver. The changes you see are to add support for Bwd direction in BN _problem  description_, which is necessary to implement & test Bwd BN solver.

Reply:
yes, I see.  Thanks Artem for clarification..